hs_core/management/utils.py

```diff
@@ -215,6 +218,48 @@ def check_relations(resource):
                       .format(resource.short_id, r.type, target))
 
 
+def fix_resourcefile_duplicates(dry_run=False, logger=None):
```

Comment:
This function really did not change how we are accessing the models.

Reply:
believe we addressed this